### PR TITLE
sqlccl: plumb csv opts through distsql spec

### PR DESCRIPTION
**Pull request by dt**

### CODE REVIEW

File: pkg/ccl/sqlccl/csv.go [hunk 1]
```diff
@@ -686,7 +686,7 @@ func importPlanHook(
 			// TODO(dt): verify db exists
 		}
 
-		comma := ','
+		var comma rune
```
Comment (benesch) [inline pkg/ccl/sqlccl/csv.go]: This is just so the default is only in one place, yeah? (As in, this was technically correct before, right?) Just checking my own understanding.

File: pkg/ccl/sqlccl/csv_test.go [hunk 2]
```diff
@@ -451,11 +490,21 @@ func TestImportStmt(t *testing.T) {
...
+			// Verify correct number of rows via COUNT.
 			sqlDB.QueryRow(`SELECT count(*) FROM t`).Scan(&result)
 			if expect := expectedRows; result != expect {
 				t.Fatalf("expected %d rows, got %d", expect, result)
 			}
+
+			// Verify correct number of NULLs via COUNT.
+			sqlDB.QueryRow(`SELECT count(*) FROM t WHERE b IS NULL`).Scan(&result)
+			expectedNulls := 0
+			if strings.Contains(tc.query, "nullif") {
```
Comment (benesch) [inline pkg/ccl/sqlccl/csv_test.go]: Where's the query that sets `nullif`? I can't seem to find it.

File: pkg/ccl/sqlccl/csv.go (hunk 1 above)
Comment (dt) [inline pkg/ccl/sqlccl/csv.go]: correct, was previously initializing the override to the default, now leaving it zero which gets changed to default in the reader.

**Comment by mjibson** [comment]
LGTM after addressing nullif comment.

---

Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/17806#-:-Ks9Q2w-pDYBhGks6hG1:b98co3o)*

File: pkg/ccl/sqlccl/csv_test.go (hunk 2 above)
Comment (dt) [inline pkg/ccl/sqlccl/csv_test.go]: thanks, nice catch and it indeed looks like it too isn't quite plumbed... looking.

Comment (dt) [inline pkg/ccl/sqlccl/csv_test.go]: er, scratch that, plumbing was okay, just needed to fix up test data. RFAL!

**Comment by mjibson** [comment]
:lgtm:

---

Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/17806#-:-Ks9Wwen9JpS6xxFFs8B:bnfp4nl)*

**Review by benesch — APPROVED** [review]

**Comment by dt** [comment]
TFTRs!

---

Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/17806#-:-Ks9iV3qXGcZpou-BmE1:bz5mair)*

**Pull request merged**